pallets/ddc-validator/src/tests.rs

```diff
@@ -218,3 +225,379 @@ fn it_sets_validation_decision_with_one_validator_in_quorum() {
 		);
 	})
 }
+
+#[test]
+fn send_signal_works_as_expected() {
+	let mut t = new_test_ext();
+
+	let validator_controller = AccountId::from([0xaa; 32]);
+
+	t.execute_with(|| {
+		assert_eq!(DdcValidator::signal(), None);
+		assert_ok!(DdcValidator::send_signal(RuntimeOrigin::signed(validator_controller)));
```

Comment:
As I remember this method was used only for demo purposes. I guess we can remove it

Reply:
I think Alisher wants to keep it